build/Targets/VSL.Versions.targets

```diff
@@ -61,14 +61,13 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <VSSemanticVersion>15.0.1</VSSemanticVersion>
+    <VSSemanticVersion>15.0</VSSemanticVersion>
```

Comment:
Ok looks like this variable was bumped up unnecessarily before. I verified from the drop share that the vsix version is still 15.0.1.xxxx and that comes from https://github.com/dotnet/roslyn-project-system/pull/1639/files#diff-ce3a3697b9bd06fb295da18808269b37R3